data/settings.js

```diff
@@ -515,123 +515,290 @@ depending on the authenticated user).`
 
 	/* Pigeonhole plugin settings. */
 
-	sieve: {
-		tags: [ 'sieve' ],
```

Comment:
Check also managesieve settings. managesieve_logout_format has old-style %variables. managesieve_max_line_length is SIZE type. managesieve_notify_capability & managesieve_sieve_capability are BOOLLIST type. Also I noticed managesieve_client_workarounds should be BOOLLIST -> DOV-7281.

Reply:
> Check also managesieve settings. managesieve_logout_format has old-style %variables.

So does IMAP

Reply:
Fixed it for ManageSieve

Reply:
Looks like there are lots of unconverted %variables.